doc/auth.md

```diff
@@ -417,17 +417,16 @@ Key | Value
 --- | ---
 Content-Type | application/json
 Accept | application/json
+Email | application/json
```

Comment:
The example value of Email is rijal.asep.nugroho@gmail.com, not application/json